client/rpc-java-client/src/main/java/org/sonarsource/sonarlint/core/rpc/client/SloopLauncher.java

```diff
@@ -122,6 +92,27 @@ private Sloop execute(Path distPath, @Nullable Path jrePath) throws IOException
     return new Sloop(serverProxy, process);
   }
 
+  private List<String> createCommand(Path distPath, @Nullable Path customJreHomePath) {
+    var jreHomePath = customJreHomePath == null ? distPath.resolve("jre") : customJreHomePath;
```

Comment:
I think we need to account that on Windows the java executable  has `.exe` exension so ` distPath.resolve("jre.exe")`

Reply:
True, and the tests were failing because of that. Fixed it